examples/pytorch/gin/README.md

```diff
@@ -4,80 +4,23 @@ Graph Isomorphism Network (GIN)
-
-```bash
-python main.py
-```
-
-An experiment on the GIN in customized settings can be run with
-```bash
-python main.py [--device 0 | --disable-cuda] --dataset COLLAB \
-               --graph_pooling_type max --neighbor_pooling_type sum
-```
-add `--degree_as_nlabel` to use one-hot encodings of node degrees as node feature vectors
-
-Results
 -------
 
-results may **fluctuate**, due to random factors and the relatively small data set. if you want to follow the paper's setting, consider the script below.
-
+Run with the following for bioinformatics graph classification (available datasets: MUTAG (default), PTC, NCI1, and PROTEINS)
 ```bash
-# 4 bioinformatics datasets setting graph_pooling_type=sum, the nodes have categorical input features 
-python main.py --dataset MUTAG --device 0  \
-                --graph_pooling_type sum --neighbor_pooling_type sum --filename MUTAG.txt
-
-python main.py --dataset PTC --device 0  \
-                --graph_pooling_type sum --neighbor_pooling_type sum --filename PTC.txt
-
-python main.py --dataset NCI1 --device 0  \
-                --graph_pooling_type sum --neighbor_pooling_type sum --filename NCI1.txt
-
-python main.py --dataset PROTEINS --device 0  \
-                --graph_pooling_type sum --neighbor_pooling_type sum --filename PROTEINS.txt
-
-# 5 social network datasets setting graph_pooling_type=mean, for the REDDIT datasets, we set all node feature vectors to be the same 
-# (thus, features here are uninformative); for the other social networks, we use one-hot encodings of node degrees.  
-python main.py --dataset COLLAB --device 0  \
-                --graph_pooling_type mean --neighbor_pooling_type sum --degree_as_nlabel --filename COLLAB.txt
-
-python main.py --dataset IMDBBINARY --device 0  \
-                --graph_pooling_type mean --neighbor_pooling_type sum --degree_as_nlabel --filename IMDBBINARY.txt
-
-python main.py --dataset IMDBMULTI --device 0  \
-                --graph_pooling_type mean --neighbor_pooling_type sum --degree_as_nlabel --filename IMDBMULTI.txt
-
-python main.py --dataset REDDITBINARY --device 0  \
-                --graph_pooling_type mean --neighbor_pooling_type sum --filename REDDITBINARY.txt --fold_idx 6 --epoch 120
-
-python main.py --dataset REDDITMULTI5K --device 0  \
-                --graph_pooling_type mean --neighbor_pooling_type sum --filename REDDITMULTI5K.txt
+python3 train.py --dataset MUTAG
 ```
 
-one fold of 10 result are below.
+> **_NOTE:_**  Users may observe results fluctuate due to the randomness of relatively small dataset.  In consistence with the original [paper ](https://arxiv.org/abs/1810.00826), five social network datasets, 'COLLAB', 'IMDBBINARY' 'IMDBMULTI' 'REDDITBINARY' and 'REDDITMULTI5K', are also available as the input. Users are encouraged to update the script slightly for social network applications, for example, replacing sum readout on bioinformatics datasets to mean readout on social datasets.
```

Comment:
Perhaps you also want to mention "--degree_as_nlabel"

Reply:
This example is implemented more for bio datasets, where one-hot encoding is always disabled.
Now, I mention `degree_as_nlabel` in the user note, to remind users when applying to soical datasets.